projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java

```diff
@@ -16,13 +13,19 @@
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.Date;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
 
 import org.apache.commons.csv.CSVFormat;
 import org.apache.commons.csv.CSVParser;
 import org.apache.commons.csv.CSVRecord;
 
 class DataHandler {
 
+  private static final String ISRAEL_RAIL_AGENCY_ID = "2";
```

Comment:
The company is called Israel Railways, so let's change to:
`s/ISRAEL_RAIL_AGENCY_ID/ISRAEL_RAILWAYS_AGENCY_ID`